tests/trilinos/vector_reference_01.cc

```diff
@@ -40,56 +40,27 @@ void test ()
   TrilinosWrappers::internal::VectorReference c (v(2));
 
   // copy VectorReference objects. note that operator= for these
-  // objects does not copy the*content* of the reference object, but
+  // objects does not copy the *content* of the reference object, but
   // assigns the value of the vector element referenced on the right
   // to the vector element referenced on the left
+  // So the applied operations result in the following actions:
+  // 1. assign c to point to where a points (c points to entry 0)
+  // 2. assign c to point to where b points (c points to entry 1)
+  // 3. assign b to point where a points (b points to entry 0)
```

Comment:
pedantic comment fixes:
1. We have a sentence fragment, a sentence without a period, and another sentence fragment. It would be nice to make this block of text more cohesive
2. I forgot the 'to point *to*' in 3 when I wrote my original comment